llm/gpt-oss-finetuning/gpt-oss-120b-lora.yaml

```diff
@@ -15,15 +24,18 @@ setup: |
   uv pip install "trl>=0.20.0" "peft>=0.17.0" "transformers>=4.55.0"
   uv pip install deepspeed
   uv pip install git+https://github.com/huggingface/accelerate.git@c0a3aefea8aa5008a0fbf55b049bd3f0efa9cbf2
+  uv pip install wandb
 
   uv pip install nvitop
 
 run: |
+  export WANDB_RUN_ID=$SKYPILOT_TASK_ID
+  export WANDB_NAME=run-$SKYPILOT_TASK_ID
   source ~/training/bin/activate
 
   MASTER_ADDR=$(echo "$SKYPILOT_NODE_IPS" | head -n1)
   NP=$(($SKYPILOT_NUM_GPUS_PER_NODE * $SKYPILOT_NUM_NODES))
 
-  python /sft/train.py --model_id openai/gpt-oss-120b --enable_lora
+  python /sft/train.py --model_id openai/gpt-oss-120b --enable_lora --resume_from_checkpoint
```

Comment:
just to confirm - if the bucket file_mounts above is not specified, this line still works, right?